data/completions/meson.build

```diff
@@ -0,0 +1,13 @@
+if get_option('bash_completion')
+  install_data(
+    'gpaste-client',
+    install_dir: join_paths(get_option('datadir'), 'bash-completion', 'completions'),
+  )
+endif
+
+if get_option('zsh_completion')
+  install_data(
+    '_gpaste-client',
+    install_dir: join_paths(get_option('datadir'), 'zsh', 'site-functions'),
```

Comment:
Might be worth adding some `install_bash_completion` and `install_zsh_completion` upstream.
Sounds like a pretty common action and would avoid having to hardcode pats in all projects

Reply:
Sure, but I'd rather not touch the internals of meson, I don't like messing with Python.